dice-where-downloader-lib/src/main/java/technology/dice/dicewhere/downloader/source/FileSource.java

```diff
@@ -5,7 +5,8 @@
 import technology.dice.dicewhere.downloader.destination.FileAcceptor;
 
 public interface FileSource {
+
   FileInfo fileInfo();
 
-  MD5Checksum produce(FileAcceptor consumer);
+  MD5Checksum produce(FileAcceptor consumer, boolean noMd5Check);
```

Comment:
Is `noMd5Check` always `false`? Also I'd name it `skipMd5Check

Reply:
No, it's a flag passed in on the execution.

I agree it should be renamed to something better to avoid double negation as @styxtwo suggested before.